Sources/Jaeger/Jaeger/Agent/CoreDataAgent.swift

```diff
@@ -55,13 +55,11 @@ public final class CoreDataAgent<RawSpan: SpanConvertible>: Agent {
      
      - Parameter config: The configuration for the core data stack and agent.
      - Parameter sender: The point of entry to report spans to a collector.
-     - Parameter objectModelBundle: The bundle where the OTCoreDataAgent.mom file was copied.
      */
-    public convenience init(config: CoreDataAgentConfiguration, sender: SpanSender, objectModelBundle: Bundle = .main) {
-        guard let modelURL = objectModelBundle.url(forResource: Constants.modelName, withExtension: "mom"),
-            let model = NSManagedObjectModel(contentsOf: modelURL) else { fatalError() }
-        let storeType: CoreDataStack.StoreType = .sql
+    public convenience init(config: CoreDataAgentConfiguration, sender: SpanSender) {
 
+        let storeType: CoreDataStack.StoreType = .sql
+        let model = AgentCoreDataObjectModel.model
         if let url = config.coreDataFolderURL {
             let stack = CoreDataStack(modelName: Constants.modelName, folderURL: url, model: model, type: storeType)
```

Comment:
is `Constants.modelName` needed anymore here?

Reply:
Done :)